### ISSUE TITLE
Rev 2 MoonratII Boards and Assemblies Order Information Capture 20240307

### ISSUE THREAD

**Issue by melanielaporte** [PubInv/moonrat#234]
1. Our previous order #W20231130095 required an additional $25.29 for the v-cut.
This order, we have selected 2 boards, Paneled by Customer. 
We would like the PCBs and assemblies to be similar to our previous order 
2. MF12 (1x4 header) should be soldered to J2 (OLED display) 
3. MF 4-6, 8 (round nuts) mounted to OLED display 
    
If additional payment required, please advise.

**Comment by melanielaporte** [comment]
Things necessary to make a successful order not obvious to a beginner 
Uploading the .... files, Selected Research development
Friday - do a dummy order, screen capture each step @melanielaporte

**Comment by melanielaporte** [comment]
JLCPCB email
![Screenshot Capture - 2024-03-16 - 08-14-42](https://github.com/PubInv/moonrat/assets/107901386/9d67227c-58dd-48b4-a932-abbd34514abd)

**Comment by melanielaporte** [comment]
Our response
![Screenshot Capture - 2024-03-16 - 08-15-48](https://github.com/PubInv/moonrat/assets/107901386/54043099-f0cf-4a71-8671-b9ad829429b4)
<img width="1728" alt="Screenshot 2024-03-15 at 7 55 39 PM" src="https://github.com/PubInv/moonrat/assets/107901386/a0682b5b-ad88-413a-92f0-1b22756bda3f">

**Comment by melanielaporte** [comment]
JLCPCB email 20240316
<img width="400" alt="Screenshot 2024-03-16 at 8 22 02 AM" src="https://github.com/PubInv/moonrat/assets/107901386/e3e06463-c7e7-4c65-9af8-2bf57153f297">

**Comment by melanielaporte** [comment]
@ForrestErickson

**Comment by melanielaporte** [comment]
Issue with JLCPCB order: due to the PCB being ordered Economy instead of Standard board, added the v-cut is not possible. 

@ForrestErickson There are 2 options. Which should we choose?
1. Go forward with the Economy PCBA Rev 2 order, requiring the GDL team to cut the board manually.
2. Cancel the Economy {CBA Rev 2 order and purchased a Standard Rev 2 PCBA today April 21st.  
![Screenshot Capture - 2024-03-21 - 11-47-01](https://github.com/PubInv/moonrat/assets/107901386/c08f2106-e8e4-4923-9db5-6650e039e1c0)

**Comment by melanielaporte** [comment]
Doing it right now, will keep the team informed.

**Comment by melanielaporte** [comment]
@ForrestErickson The site is down tonight, I will order in the morning, 03/23/2024.
![Screenshot Capture - 2024-03-22 - 21-14-27](https://github.com/PubInv/moonrat/assets/107901386/428de7d4-1f12-4098-bd28-b5e60a8b4c38)

**Comment by melanielaporte** [comment]
New PCBA order has been placed, invoice sent.

![Screenshot Capture - 2024-03-24 - 16-34-04](https://github.com/PubInv/moonrat/assets/107901386/17607ec1-da9a-42a2-8316-607be87575de)
@RobertLRead @ForrestErickson

**Comment by ForrestErickson** [comment]
@melanielaporte 
Can you summarized here the cost delta so we remember? IIRC it was not much but we should know that the "V" cut is a distinct cost in addition to the component and board and assembly costs. 
Can you please  itemized this cost in the Hardware X article too?

If we had plenty of free time we could even make a copy of the MoonRatII schematic and PCB and break into two schematics and two PCBs. One each for the Controller and the Heater assembly and then work up an order with JLCPCB for the two assemblies seperate. Another order with the two assemblies "Panelized" to further understand the cost implications. 


CC: @RobertLRead

**Comment by melanielaporte** [comment]
JLCPCB production is finally a GO!

![Screenshot Capture - 2024-03-25 - 22-05-57](https://github.com/PubInv/moonrat/assets/107901386/966edd4b-547c-4724-8ed5-eb0c5fc61142)
